examples/fram_spi_simpletest.py

```diff
@@ -0,0 +1,29 @@
+## Simple Example For CircuitPython/Python SPI FRAM Library
+
+import board
+import busio
+import digitalio
+import adafruit_fram
+
+## Create a FRAM object.
+spi = busio.SPI(board.SCK, board.MOSI, board.MISO)
+cs = digitalio.DigitalInOut(board.D5)
```

Comment:
**Personal Post-It**

carter mentioned this:
> I changed CS in your example to 7, wanted to avoid using the shifted pin 5 output on Itsy

Reply:
FYI - You can leave it as D5. I think that's the accepted CS standard. I just like using an Itsy for dev work and just wanted to be explicit about how I got it working with the wiring shown in my setup photo.

Reply:
Ok. I did grab D5 usage from elsewhere, but I do think you make a good point.

Reply:
I would stick with D5 - it's what we use most of our examples.